src/airport.js

```diff
@@ -1 +1,49 @@
+class Airport {
+    hangar;
+    #maxCapacity;
+    weather;
 
+    constructor(plane = [], capacity = 3, weather = 'Clear') {
+        this.hangar = plane;
+        this.#maxCapacity = capacity;
+        this.weather = weather;
+    }
```

Comment:
www - implemented at least one private property.
EBI - wasn't able to make certain functions work when all properties were private, especially when trying to access properties from the plane class through the airport. 
Goals - to learn more about private properties and when to implement them, in addition to how it can work between classes.